sdk/communication/Azure.Communication.Chat/tests/Infrastructure/ChatRecordedTestSanitizer.cs

```diff
@@ -21,20 +24,23 @@ public override void SanitizeHeaders(IDictionary<string, string[]> headers)
         {
             if (headers.ContainsKey(HttpHeader.Names.Authorization))
             {
-                if (headers.ContainsKey(HttpHeader.Names.UserAgent) && headers[HttpHeader.Names.UserAgent].Any(x => x.Contains("Communication.Chat")))
+                if (headers.ContainsKey(HttpHeader.Names.UserAgent) && headers[HttpHeader.Names.UserAgent].Any(x=>x.Contains("Communication.Chat")))
                 {
-                    headers[HttpHeader.Names.Authorization] = new[] { SanitizedUnsignedUserTokenValue };
+                    headers[HttpHeader.Names.Authorization] = new[] { SanitizedChatAuthHeaderValue };
                     return;
                 }
                 headers[HttpHeader.Names.Authorization] = new[] { SanitizeValue };
             }
         }
+
         public override string SanitizeVariable(string variableName, string environmentVariableValue)
-            => variableName switch
```

Comment:
changed to expression body before, any reason for changing back?

Reply:
done